daprdocs/content/en/getting-started/install-dapr-cli.md

```diff
@@ -32,6 +32,12 @@ Verify by opening Explorer and entering `C:\dapr` into the address bar. You shou
 {{% /codetab %}}
 
 {{% codetab %}}
+
+> **Note:** For M1 Macs, homebrew is not supported. You will need to use the dapr install script and have the rosetta amd64 compatibility layer installed. If you do not have it installed already, you can run the following:
```

Comment:
Comment looks good. Although please use alert shortcode and move to end of codetab:

{{% alert title="Note for M1 Macs" color="primary" %}}
Homebrew is not currently supported on M1 Macs. You will need to use the dapr install script and have the rosetta amd64 compatibility layer installed. If you do not have it installed already, you can run the following:

```bash
softwareupdate --install-rosetta
```
{{% /alert %}}

Reply:
Cool, will do.